tests/e2e/assistant.spec.js

```diff
@@ -1,12 +1,21 @@
-import { test, expect } from './fixtures';
-
+import {
+  test,
+  expect
+} from './fixtures';
 
 const MediaType = {
   video: "video",
   image: "image",
   none: "none",
 };
 
+const LinkResult = {
```

Comment:
These icons will be removed in PR #640 - for the future, the contents of `sourceTypes` could be used instead? https://github.com/AFP-Medialab/verification-plugin/blob/609f9710c4edc73fcd23df9616a63a252787a0a7/src/components/NavItems/Assistant/AssistantScrapeResults/AssistantSCResults.jsx#L48